imap_processing/cdf/config/imap_ultra_l1c_variable_attrs.yaml

```diff
@@ -17,6 +17,11 @@ default_float32_attrs: &default_float32
   VALIDMAX: 3.4028235e+38
   dtype: float32
 
+energy_dependent_attrs: &energy_dependent
+  <<: *default_float32
+  DEPEND_0: energy_bin_geometric_mean
```

Comment:
I have been told my @vmartinez-cu that DEPEND_0 must be epoch / time varying. So this should be DEPEND_1 instead I think.